### PR TITLE
Update metricflow-commands.md

### PR DESCRIPTION
**Pull request by mirnawong1**
add gotcha

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/build/metricflow-commands.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ You can install [MetricFlow](https://github.com/dbt-labs/metricflow#getting-star
 
 </Tabs>
 
+Something to note, MetricFlow `mf` commands return an error if you have a Metafont latex package installed. To run `mf` commands, uninstall the package.
```
Comment (mirnawong1) [inline website/docs/docs/build/metricflow-commands.md:63]: hey @Jstein77 , does this also apply to dbt cloud cli `dbt sl` commands too?

File: website/docs/docs/build/metricflow-commands.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ MetricFlow is a dbt package that allows you to define and query metrics in your
 
 MetricFlow commands are embedded in the dbt Cloud CLI, which means you can immediately run them once you install the dbt Cloud CLI. 
 
-A benefit to using the dbt Cloud CLI or dbt Cloud IDE is that you won't need to manage versioning &mdash; your dbt Cloud account will automatically manage the versioning for you.
+A benefit to using the dbt Cloud CLI or dbt Cloud IDE is that you won't need to manage versioning &mdash; your dbt Cloud account will automatically manage the versioning.
```
Comment (Jstein77) [inline website/docs/docs/build/metricflow-commands.md:28]: You can't run metricflow commands in the ide just yet.

File: website/docs/docs/build/metricflow-commands.md (hunk 1 above)
Comment (Jstein77) [reply website/docs/docs/build/metricflow-commands.md:63]: Yea, i think the dependencies are the same.

**Review by Jstein77 — APPROVED** [review]
LGTM! Left one comment.

**Pull request merged**